lib/commands/project/create.ts

```diff
@@ -0,0 +1,70 @@
+///<reference path="../../.d.ts"/>
+"use strict";
+
+import Future = require("fibers/future");
+import ProjectCommandBaseLib = require("./project-command-base");
+import path = require("path");
+import options = require("./../../options");
+
+export class CreateCommand extends ProjectCommandBaseLib.ProjectCommandBase {
+	private static GENERATOR_NAME = "generator-kendo-ui-mobile";
+
+	constructor(public $project: Project.IProject,
+				public $errors: IErrors,
+				public $logger: ILogger,
+		private $childProcess: IChildProcess,
+		private $projectConstants: Project.IProjectConstants,
+		private $appScaffoldingExtensionsService: IAppScaffoldingExtensionsService,
+		private $screenBuilderService: IScreenBuilderService) {
+		super($project, $errors, $logger);
+	}
+
+	public execute(args: string[]): IFuture<void> {
+		return (() => {
+			try {
+				this.$screenBuilderService.prepareScreenBuilder(CreateCommand.GENERATOR_NAME).wait();
+				var appScaffoldingPath = this.$appScaffoldingExtensionsService.appScaffoldingPath;
+				var projectDirPath = path.resolve(options.path || ".");
+
+				var cliServicePath = path.join(appScaffoldingPath, "lib/cliService");
+				var Scaffolder = require(cliServicePath);
+				var connector = {
+					generatorsCache: path.join(appScaffoldingPath, "cache"),
+					path: projectDirPath,
+					dependencies: ['generator-kendo-ui-mobile@latest'],
+					connect: function _cnnct(done: any) {
+						done();
+					},
+					logger: this.$logger.trace.bind(this.$logger)
+				};
+				var s = new Scaffolder(connector);
+
+				try {
+					var future = new Future<void>();
+
+					s.promptGenerate("app", (err:any) => {
+						if (err) {
+							this.$logger.trace(err);
+							future.throw(err);
+						} else {
+							future.return();
+						}
+					});
+
+					future.wait();
+				} catch(e) {
+
+				}
+
+				this.$logger.trace("Installing bower");
+				this.$childProcess.exec("bower install", { cwd: projectDirPath }).wait();
+
+				this.initializeProjectFromExistingFiles(this.$projectConstants.TARGET_FRAMEWORK_IDENTIFIERS.Cordova).wait();
```

Comment:
Because this files are generated from kendo-ui-generator. We want to create appbuilder project and just init in directory with already created files is enough.